source/changelog.txt

```diff
@@ -12,6 +12,20 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.2
+------
+
+*Released October 8, 2024*
+
+Contains internal enhancements and improvements.
+
+Fixes startup performance regression introduced in ``v2.3.1``:
+
+- :issue:`MONGOSH-1878` - Use TLS ``allowPartialTrustChain`` flag
```

Comment:
This item is not user-facing, apart from the startup performance improvements that come along with it. I'd remove the detailed description here.

If you want to, feel free to include these two items, since they will actually be visible to users (just not affect a lot of them):

- [[MONGOSH-1447](https://jira.mongodb.org/browse/MONGOSH-1447)] - getShardDistribution does not support sharded time series collections
- [[MONGOSH-1776](https://jira.mongodb.org/browse/MONGOSH-1776)] - Hide startup warnings for local dev environments

The latter refers to "local Atlas" environments started by the Atlas CLI, the former fixes the bug (i.e. now  `getShardDistribution()` _does_ support time series collections)

Reply:
Updated to remove the `:issue:` and detailed description for `MONGOSH-1878`.

I am comfortable leaving out the 2 additional bug fixes as they are still available in the full JIRA release notes.

If you decide you would prefer them included, please let me know and I will add them ASAP.